base/div.jl

```diff
@@ -66,8 +66,8 @@ without any intermediate rounding.
   ``(y, 0]`` otherwise. The result may not be exact if `x` and `y` have different signs, and
   `abs(x) < abs(y)`. See also [`RoundDown`](@ref).
 
-- if `r == RoundUp`, then the result is in the interval `(-y,0]` if `y` is positive, or
-  `[0,-y)` otherwise. The result may not be exact if `x` and `y` have the same sign, and
+- if `r == RoundUp`, then the result is in the interval `(-y, 0]` if `y` is positive, or
+  `[0, -y)` otherwise. The result may not be exact if `x` and `y` have the same sign, and
```

Comment:
```suggestion
- if `r == RoundUp`, then the result is in the interval ``(-y, 0]`` if `y` is positive, or
  ``[0, -y)`` otherwise. The result may not be exact if `x` and `y` have the same sign, and
```